docs/modules/ROOT/pages/how-tos/quick-start.adoc

```diff
@@ -0,0 +1,66 @@
+= Quick start
+
+This how-to will guide you to a working Ceph setup using the default configuration.
```

Comment:
Shouldn't this page be in the tutorial category? Sorry if I missed that one in the last review.

Reply:
Yeah, probably makes more sense as a tutorial